test_sets/SDL5_0/rc_CLIMATE_RADIO.txt

```diff
@@ -130,3 +130,4 @@
 ./test_scripts/RC/CLIMATE_RADIO/SetInteriorVehicleData/025_UNSUPPORTED_RESOURCE_climateEnable_false.lua
 ./test_scripts/RC/CLIMATE_RADIO/SetInteriorVehicleData/026_RADIO_Negative_try_to_set_readOnly_availableHdChannels.lua
 ./test_scripts/RC/CLIMATE_RADIO/SetInteriorVehicleData/027_UNSUPPORTED_RESOURCE_availableHdChannelsAvailable_false.lua
+./test_scripts/RC/CLIMATE_RADIO/SetInteriorVehicleData/028_SUCCESS_cut_off_params_for_another_module.lua
```

Comment:
Please add script to `Defects_releases_4_x_and_5_x.txt` test set since issue `2664` had been fixed in scope of `5.0.0` release